src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -550,8 +550,18 @@ public static Activity createActivity(DataObject gameJson)
             }
         }
 
+        RichPresence.Emoji emoji = null;
+        if (!gameJson.isNull("emoji"))
```

Comment:
Maybe add "emoji" to the `richGameFields` as well?

https://github.com/DV8FromTheWorld/JDA/blob/aba40e211af889796b3d431b6a84458a14a0cdde/src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java#L61-L74